codespell_lib/data/dictionary.txt

```diff
@@ -36398,6 +36479,8 @@ workes->works
 workfow->workflow
 workfows->workflows
 workign->working
+workled->worklet, worked,
+workleds->worklets, worked,
```

Comment:
I am not favour of this entry. This one is more like a CamelCase for workLEDs, workLeds. 
Looking at the search results seem valid coding.
Splitting `work` and `LEDs` will also break source code.
Personally, I would drop this one.

* https://github.com/search?q=workled&type=code  (234)
* https://github.com/search?q=workleds&type=code  (10)